pkg/kudoctl/cmd/package_verify.go

```diff
@@ -50,10 +50,10 @@ func (c *packageVerifyCmd) run(fs afero.Fs, path string) error {
 	}
 	if errors != nil {
 		printErrors(c.out, errors)
-		return fmt.Errorf("operator verification errors: %v", len(errors))
+		return fmt.Errorf("package verification errors: %v", len(errors))
 	}
-	if warnings == nil && errors == nil {
-		fmt.Fprintf(c.out, "operator is valid\n")
+	if warnings == nil {
```

Comment:
this is incorrect.   This should be if errors == nil 
```suggestion
	if errors == nil {
```